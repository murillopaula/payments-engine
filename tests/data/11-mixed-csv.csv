type,client,tx,amount
deposit,11,1101,1.1234
deposit, 12, 1102, 2.5678
withdrawal,  11,  1103,  0.1234
dispute,12,1102
deposit,11,1104,5.0000
resolve,12,1102,
withdrawal,11,1105
withdrawal,12,1106,0.5678
