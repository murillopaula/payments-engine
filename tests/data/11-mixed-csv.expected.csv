client,available,held,total,locked
11,6.0000,0.0000,6.0000,false
12,2.0000,0.0000,2.0000,false
